### ISSUE TITLE
Is this project compatible with neutrino?

### ISSUE THREAD

**Issue by Emad-salah** [alexbosworth/ln-service#90]
Hi everyone,

I'm currently trying to get ln-service running but I've been constantly getting this error: `[ 503, 'FailedToDetermineWalletUnlockStatus', [ 400, 'ExpectedLndForGetInfoRequest' ] ]`

These are the arguments that I'm using to run lnd:
`lnd --bitcoin.active --bitcoin.testnet --debuglevel=debug --bitcoin.node=neutrino --neutrino.connect=faucet.lightning.community`

And here's everything that I'm using and their respective versions:
lnd - v0.6.1-beta-174
go - v1.12.5
node - v8.11.4

Looking forward to getting ln-service running on my machine!

Thanks!

**Comment by alexbosworth** [comment]
How are you calling ln-service?

I can try and improve the documentation about getting it going, it should work with Neutrino

**Comment by Emad-salah** [comment]
Thanks for the quick reply! I'm running ln-service in REST mode, I've created an env file in the project that's close to the one in the documentation with this content:

```
GRPC_SSL_CIPHER_SUITES='HIGH+ECDSA'
LNSERVICE_CHAIN='bitcoin'
LNSERVICE_LND_DIR='C:/Users/Admin/AppData/Local/Lnd'
LNSERVICE_NETWORK='testnet'
LNSERVICE_SECRET_KEY='REPLACE!WITH!SECRET!KEY!'
```

and then ran ln-service using: `yarn start`

**Comment by alexbosworth** [comment]
I would try the grpc mode instead of the REST server mode, I intend to split the REST server mode out actually

**Comment by Emad-salah** [comment]
Sure, will try it out shortly and let you know if it works fine. Thanks!

**Comment by alexbosworth** [comment]
I actually see there is an error that is causing this, it is a bug

**Comment by alexbosworth** [comment]
This should get fixed in the next release

**Comment by Emad-salah** [comment]
Thanks a lot! I've tried out using the grpc mode and it worked fine. Looking forward to checking out the patched ln-service! 😊

**Comment by alexbosworth** [comment]
Should be fixed in 39.0.0
